common/src/main/java/com/wynntils/core/components/Models.java

```diff
@@ -105,6 +106,7 @@ public final class Models {
     public static final SeaskipperModel Seaskipper = new SeaskipperModel(Item);
     public static final ShamanMaskModel ShamanMask = new ShamanMaskModel(WorldState);
     public static final ShamanTotemModel ShamanTotem = new ShamanTotemModel(WorldState);
+    public static final ShinyModel Shiny = new ShinyModel();
```

Comment:
Why is this in the models with dependencies list but has no dependency, should it depend on the Item model?

Reply:
It should. Nice catch. Ill push a fix.